classes/markdown.js

```diff
@@ -0,0 +1,26 @@
+import commonmark from 'commonmark';
+
+/**
+ * Class for generating HTML from markdown messages
+ * 
+ * @param   {string} input - Message body containing markdown
+ */
+export default class Markdown {
+    constructor(input) {
+        // Replace < to escape HTML
+        this.input = input.replace(/</g, '&lt;');
```

Comment:
`input` is never actually used outside the constructor, right? No need to store it on the object?

Reply:
Yeah, I was thinking of possibly adding more features to class (similar to matrix-react-sdk's own `Markdown` class) but later decided against it. I'll clean it up.